projects/demo-playwright/tests/demo/source-code-link.pw.spec.ts

```diff
@@ -6,6 +6,7 @@ test.describe('Source code button', () => {
     const proprietary = process.env['PROPRIETARY'] === 'true';
     const currentBranch =
         process.env['GITHUB_HEAD_REF'] || process.env['GITHUB_REF_NAME'] || 'main';
+    const repository = process.env['GITHUB_REPOSITORY'] || 'taiga-family/taiga-ui';
```

Comment:
<img width="674" height="63" alt="docs" src="https://github.com/user-attachments/assets/2c80262f-1078-43e4-99a7-7cba7498f4d5" />

https://docs.github.com/en/actions/reference/workflows-and-actions/variables